### PR TITLE
add r-marquee

### PR DESCRIPTION
**Pull request by vitodelgrec**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-marquee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    - {{ cran_mirror }}/src/contrib/Archive/marquee/marquee_{{ version }}.tar.gz
+  sha256: ef4b3798d1a85b0898eb03d8a8a4ecc3addebdceefacfb022c8ca37989b51994
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```
Comment (mfansler) [inline recipes/r-marquee/meta.yaml:21]: Common DSO whitelist:

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'        # [win]
    - '*/Rblas.dll'    # [win]
    - '*/Rlapack.dll'  # [win]

```

File: recipes/r-marquee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```
Comment (mfansler) [inline recipes/r-marquee/meta.yaml:26]: Need different stdlib for win:

```suggestion
    - {{ compiler('c') }}        # [not win]
    - {{ stdlib('c') }}          # [not win]
    - {{ compiler('m2w64_c') }}  # [win]
    - {{ stdlib('m2w64_c') }}    # [win]
```

File: recipes/r-marquee/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    - r-textshaping >=0.4.0
+    - r-vctrs
+
+test:
+  commands:
+    - $R -e "library('marquee')"           # [not win]
+    - "\"%R%\" -e \"library('marquee')\""  # [win]
+
+about:
+  home: https://marquee.r-lib.org, https://github.com/r-lib/marquee
```
Comment (mfansler) [inline recipes/r-marquee/meta.yaml:65]: Split URLs

```suggestion
  home: https://marquee.r-lib.org
  dev_url: https://github.com/r-lib/marquee
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Some suggested changes to get the win version passing.

**Comment by vitodelgrec** [comment]
thank you for the review suggestions 🙏

@conda-forge/help-r

**Review by mfansler — APPROVED** [review]

**Pull request merged**